### PR TITLE
CSS updates, style tuning of Embedded component and Annotation Component

### PR DESCRIPTION
**Pull request by nickhsine**
@garfieldduck

### CODE REVIEW

**Comment by garfieldduck** [comment]
LGTM :+1:

**Comment by nickhsine** [comment]
@garfieldduck 有新的commit ，再幫忙看一下。

**Comment by nickhsine** [comment]
@garfieldduck 
please have a see the Annotation component.

File: src/components/article/Common.scss [hunk 1]
```diff
@@ -11,15 +11,25 @@ $red: $harley-davidson-orange;
   margin: 0 rem(24px) 0 rem(24px);
 }
 
+.disable-inner-block {
+  margin: 0 rem(-24px) 0 rem(-24px);
```
Comment (garfieldduck) [inline src/components/article/Common.scss]: do you want to put the margin as a variable?

**Comment by garfieldduck** [comment]
LGTM :+1:  :+1:  :+1:

**Pull request merged**